cpp/arcticdb/toolbox/library_tool.hpp

```diff
@@ -54,6 +61,7 @@ class LibraryTool {
 
 private:
     std::shared_ptr<storage::Library> lib_;
+    std::shared_ptr<arcticdb::async::AsyncStore<util::SysClock>> store_;
```

Comment:
I think `store_` can probably do everything `lib_` can do, so could get rid of `lib_`, but this is only a debug/testing tool so not a requirement

Reply:
Cool, done